kernel_crawler/oracle.py

```diff
@@ -1,70 +1,46 @@
             'http://yum.oracle.com/repo/OracleLinux/OL6/latest/' + self.arch + '/',
             'http://yum.oracle.com/repo/OracleLinux/OL6/MODRHCK/' + self.arch + '/',
             'http://yum.oracle.com/repo/OracleLinux/OL6/UEKR4/' + self.arch + '/',
             'http://yum.oracle.com/repo/OracleLinux/OL6/UEKR3/latest/' + self.arch + '/',
             'http://yum.oracle.com/repo/OracleLinux/OL6/UEK/latest/' + self.arch + '/',
-        ]
-
-    def __init__(self, arch):
-        super(Oracle6Mirror, self).__init__([], arch)
-
-    def list_repos(self):
-        return [OracleRepository(url) for url in self.repos()]
-
-    def to_driverkit_config(self, release, deps):
-        for dep in deps:
-            if dep.find("devel") != -1:
-                return repo.DriverKitConfig(release, "oracle6", dep)
-
-class Oracle7Mirror(repo.Distro):
-    def repos(self):
-        return [
+            # Oracle7
             'http://yum.oracle.com/repo/OracleLinux/OL7/latest/' + self.arch + '/',
             'http://yum.oracle.com/repo/OracleLinux/OL7/MODRHCK/' + self.arch + '/',
             'http://yum.oracle.com/repo/OracleLinux/OL7/UEKR6/' + self.arch + '/',
             'http://yum.oracle.com/repo/OracleLinux/OL7/UEKR5/' + self.arch + '/',
             'http://yum.oracle.com/repo/OracleLinux/OL7/UEKR4/' + self.arch + '/',
             'http://yum.oracle.com/repo/OracleLinux/OL7/UEKR3/' + self.arch + '/',
-        ]
-
-    def __init__(self, arch):
-        super(Oracle7Mirror, self).__init__([], arch)
-
-    def list_repos(self):
-        return [OracleRepository(url) for url in self.repos()]
-
-    def to_driverkit_config(self, release, deps):
-        for dep in deps:
-            if dep.find("devel") != -1:
-                return repo.DriverKitConfig(release, "oracle7", dep)
-
-
-class Oracle8Mirror(repo.Distro):
-    def repos(self):
-        return [
+            # Oracle8
             'http://yum.oracle.com/repo/OracleLinux/OL8/baseos/latest/' + self.arch + '/',
             'http://yum.oracle.com/repo/OracleLinux/OL8/UEKR6/' + self.arch + '/',
+            'http://yum.oracle.com/repo/OracleLinux/OL8/appstream/' + self.arch + '/',
+            # Oracle9
+            'http://yum.oracle.com/repo/OracleLinux/OL9/baseos/latest/' + self.arch + '/',
+            'http://yum.oracle.com/repo/OracleLinux/OL9/UEKR7/' + self.arch + '/',
+            'http://yum.oracle.com/repo/OracleLinux/OL9/appstream/' + self.arch + '/',
         ]
 
     def __init__(self, arch):
-        super(Oracle8Mirror, self).__init__([], arch)
+        super(OracleMirror, self).__init__([], arch)
 
     def list_repos(self):
         return [OracleRepository(url) for url in self.repos()]
 
     def to_driverkit_config(self, release, deps):
         for dep in deps:
             if dep.find("devel") != -1:
-                return repo.DriverKitConfig(release, "oracle8", dep)
\ No newline at end of file
+                return repo.DriverKitConfig(release, "ol", dep)
```

Comment:
@EXONER4TED this change should be ported also to Driverkit! 
Can we do it in https://github.com/falcosecurity/driverkit/pull/260?

Reply:
On it!